pabcnetc/ConsoleCompiler.cs

```diff
@@ -433,77 +455,81 @@ public static int Main(string[] initialArgs)
-                
+
             }
-                
 
-            for (int i = 0; i < ConsoleBufferWidth-1; i++)
-              BlankString += " ";
+
+            BlankString = new string(' ', ConsoleBufferWidth - 1);
 
             Console.ForegroundColor = ConsoleColor.White;
             OutputDirectory="";
             
             Console.Title = StringResourcesGet("STARTING");
             Reset();
             Console.Title = Compiler.Banner;
-            
-
-            //Console.CursorLeft = (Console.BufferWidth - Compiler.Banner.Length) / 2;
-            //Console.WriteLine(Compiler.Banner); 
 
 
-            short_output = args.Count != 1;
+            //Console.CursorLeft = (Console.BufferWidth - Compiler.Banner.Length) / 2;
+            //Console.WriteLine(Compiler.Banner);
 
-            if (args.Count >= 1) 
+            if (args.Count > 0)
             {
-                FileName = args[0];
-            }
-            if (args.Count >= 2)
-                if (args[1] == "/rebuildnodebug")
+                List<FileInfo> fileInfo = new List<FileInfo>();
+                for (int i = 0; i < args.Count; i++)
                 {
-                    Rebuild = true;
-                    Compiler.InternalDebug.IncludeDebugInfoInPCU = false;
-                    short_output = false;
-                    DetailOut = true;
-                }
-                else
-                    if (args[1] == "/rebuild")
+                    if (args[i] == "/rebuildnodebug")
+                    {
+                        Rebuild = true;
+                        Compiler.InternalDebug.IncludeDebugInfoInPCU = false;
+                        short_output = false;
+                        DetailOut = true;
+                    }
+                    else if (args[i] == "/rebuild")
                     {
                         Rebuild = true;
                         short_output = false;
                         DetailOut = true;
                     }
                     else
-                        TagertFileName = args[1];
-            outputType = CompilerOptions.OutputType.ConsoleApplicaton;
-            if (args.Count >= 3)
-                    SetOutType(args[2]);
-            
-
-
-            ShowConnectedParsers();
-            ShowConnectedConversions();
-            if (args.Count >= 1)
-            {
-                return CompileAssembly(FileName, outputType, true);
+                    {
+                        if (File.Exists(args[i]) || File.Exists(Environment.CurrentDirectory + args[i]))
+                        {
+                            fileInfo.Add(new FileInfo(args[i]));
+                        }
+                        else
+                        {
+                            ExecuteCommand(args[i]);
+                        }
+                    }
+                }
+                ShowConnectedParsers();
+                ShowConnectedConversions();
```

Comment:
This was also called in the `ExecuteCommand("?");` branch, before your changes...

Reply:
I cannot find call ExecuteCommand("?"); before it. I see a call to this command below, in the case of running interactive mode. If you don't mind, could you clarify this?

Reply:
I see a call to this command below, in the case of running interactive mode. If you don't mind, could you clarify this?